tests/inference/test_utils.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import torch
+
+from utils import get_deployment_actor_options, StoppingCriteriaSub, max_input_len, get_torch_dtype, is_cpu_without_ipex
+from inference_config import InferenceConfig, DEVICE_CPU
+
+# Mock the InferenceConfig for testing
+@pytest.fixture
+def mock_infer_conf():
+    infer_conf = InferenceConfig()
+    infer_conf.ipex.enabled = False
+    infer_conf.deepspeed = False
+    infer_conf.device = DEVICE_CPU
+    infer_conf.cpus_per_worker = 1
+    infer_conf.gpus_per_worker = 0
+    infer_conf.hpus_per_worker = 0
+    return infer_conf
+
+def test_get_deployment_actor_options_cpu(mock_infer_conf):
+    actor_options = get_deployment_actor_options(mock_infer_conf)
+    assert actor_options['num_cpus'] == mock_infer_conf.cpus_per_worker
+    assert 'num_gpus' not in actor_options
+    assert 'resources' not in actor_options
+
+# Add more tests for different configurations of InferenceConfig
+
+def test_stopping_criteria_sub():
+    stopping_criteria = StoppingCriteriaSub(stops=[torch.tensor([1, 2, 3])])
+    input_ids = torch.tensor([[1, 2, 3]])
+    scores = torch.tensor([0.0])
+    assert stopping_criteria(input_ids, scores) == True
```

Comment:
`assert stopping_criteria(input_ids, scores)` ?

Reply:
this is test whether the stop condition is met。what‘s wrong with it？i should add more condition such as input_ids is none, stops is none ,etc

Reply:
I mean `== True` here is meaningless. Will it make sense if we use `assert stopping_criteria(input_ids, scores)` instead?